### ISSUE TITLE
Relax Agent creation notification

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#5042]
As one cannot create an agent without some supplemental data, can we stop with all of these?

![image](https://user-images.githubusercontent.com/5725767/189751822-91bed152-d438-458e-b1fb-ed5b20baca83.png)

AND the message that you shouldn't do this that requires an extra click?

**Comment by dustymc** [comment]
There's no relationship between those two things, and https://github.com/ArctosDB/arctos/issues/5039 happened - and was caught - under the most recent architecture. There's value in the checks, and there's value in the notifications.

**Comment by Jegelewicz** [comment]
The notification does not offer any "check". It just says force creating an agent is a bad idea. If the notification said WHY any particular agent creation might be a bad idea, that would be helpful.

**Comment by Jegelewicz** [comment]
Also - I haven't create ANY agent in the last year that doesn't give me this warning - I ignore it and I'm sure others do too. It is not helpful when you know you will get it no matter what you do.

**Comment by campmlc** [comment]
Agree.

**Comment by campmlc** [comment]
Just had to force create an agent for whom I had provided an OrcID, an postdoctoral student relation of, an alive date, but not a middle initial. She apparently doesn't use one, even though I went to the effort to search her papers. The orcID should be enough. I shouldn't have to force creation. Please let's make this easier to do.

**Comment by Jegelewicz** [comment]
As it is right now it doesn't matter what you do, you get the "there may be problems" message and HAVE to force create. It is a waste of time.

**Comment by KyndallH** [comment]
I was just complaining to Angie about this! Forced created an agent yesterday with a full middle name, PhD student relationship, email, phone number, and alive on this date. I had no idea what else I needed to make it, so I didn't have to force create.

**Comment by ewommack** [comment]
Just a clarification (and @dustymc check me on this) -> just because you have to force create an agent does not mean it is going to get caught in the agent -> verbatim agent switch coming up or that it is a low quality agent. 
We're poking @dustymc on when and where we get that lovely force creation notice here. 

Maybe a really good explanation on why we keep getting that notice would help? I just keep ignoring it now...

**Comment by dustymc** [comment]
>  why we keep getting that notice 

The obvious duplicate - even created by the same person - that was in my notifications yesterday is the sort of thing that's making me hesitant to relax anything.

I don't know what to do. I suppose someone's giving excessive rights without training, but WHATEVER the cause we're going to get stuck somewhere before "awesome," no matter what technical bars I try to put up, if we don't somehow get users into the spirit of not unnecessarily creating low-information agents that just manage to squeak past the rules at the time.

>a really good explanation 

It's always a plea for more information.

>The orcID should be enough

It's absolutely not unless other users are putting that level of effort in. While you spend 20 minutes making sure the ORCID matches up with the agent and they don't already exist and such (yay you!), I can create 20 variations of the same agent using low-quality 'probably alive while collecting' dates that I pulled out of my cruddy data (not-so-yay me).

I suppose that all adds up to me being willing to lower the paranoia level of the creation check, but we desperately need to find a way to address the clear social issue at the same time or I'm afraid this is going to turn into a whole bunch of effort for still-sorta-meh agent data.

**Comment by ewommack** [comment]
> t's absolutely not unless other users are putting that level of effort in. While you spend 20 minutes making sure the ORCID matches up with the agent and they don't already exist and such (yay you!), I can create 20 variations of the same agent using low-quality 'probably alive while collecting' dates that I pulled out of my cruddy data (not-so-yay me).

Not a criticism, but could you show us what it looks like to create an agent that doesn't bring up the force an agent warning @dustymc? Maybe we should be thinking about the problem from another direction.

**Comment by dustymc** [comment]
>u show us what it looks like to create an agent that doesn't bring up the force an agent warning

I'm not sure what I can do other than point to https://github.com/ArctosDB/PG/blob/master/component/agent.cfc - I think the messages are clear, I'm happy to adjust or attempt to explain if they're not.

(I suspect most of the complaints involve dots, a check which was introduced years ago when a bunch of unabbreviated names were tossed. All of the messages originate in real problems.)

>Maybe we should be thinking about the problem from another direction.

Yes, exactly!! 

1. Some sort of independent assertion of dates and relationships is useful; they can help a future user decide if they've got stuff from the same agent.
2. Reasserting things that can easily be derived from unverified data - an agent was alive when they collected, or collecting establishes some relationship with a repository - just pollutes everything by introducing low-quality data dressed up as high-quality data.

I think this issue (and many like it) demonstrate that getting (1) and rejecting (2) is not entirely a technical problem, although there are certainly technical aspects. Arctos is pretty good at name variants (Jim/James and such), for example.

Along with not losing any information, skipping the creation process and just using verbatim saves a lot of work. It's hard for me to imagine anyone would be doing this if they understood what we're up to and what it means for the usability of their data (but then I have trouble imagining all sorts of reality...). 

While there's definitely a lot less (2) than there used to be, and agents devoid of even recycled data aren't being created at all, the logs suggest that we still have quite a bit of (2) happening, and that we need - uhhh, something. Communication, education, ?????????????????????

**Comment by ewommack** [comment]
>  we need - uhhh, something. Communication, education, ?????????????????????

So perhaps lets try education. Instead of a warning for almost every agent that we create (which a lot of us ignore), maybe a page we have to click through to get to creating agents that says "watch out! these are shared resources and it is really easy to create bad data. How do you avoid that? Make sure you've read your documentation before you go forward: -with links to documentation-"
?

**Comment by KyndallH** [comment]
For the agent I forced yesterday - there were no dots. Did I need start and end dates on the email address or student of? Maybe highlight the problems cause I scrolled up and had no clue what else needed to be added. Other websites will highlight the fields that need/should be populated. 

<img width="756" alt="Screen Shot 2022-12-15 at 9 24 49 AM" src="https://user-images.githubusercontent.com/16887896/208208314-c89b0f6a-4955-47ca-9469-a1fa758219e0.png">

<img width="1263" alt="Screen Shot 2022-12-15 at 9 24 56 AM" src="https://user-images.githubusercontent.com/16887896/208208328-2bd44e7c-dd72-4e13-97be-e0cec994cc79.png">

<img width="1456" alt="Screen Shot 2022-12-15 at 9 25 00 AM" src="https://user-images.githubusercontent.com/16887896/208208335-573fd33c-529f-4255-8bf3-28742488a4f8.png">

**Comment by dustymc** [comment]
>that says

I think the current page is all from agent committee but it's easy enough to change text if you have better ideas.

>no dots.

No, there weren't - and no message. Maybe the thing's just broken, or broken for some users?? I'll dig ASAP, thanks for the screenshot, those are super useful.


>start and end dates o

Nope, for those the check is just exist or not, and it'll just outright refuse if they're not there. That should be clarified.

The message-but-allow-force is just from the namestring (also probably needs clarified), and should be informative- I tried to create Dustin McDonald and....


<img width="790" alt="Screenshot 2022-12-16 at 3 48 56 PM" src="https://user-images.githubusercontent.com/5720791/208209365-5c261158-0658-4f1e-bf7a-b4426672b3b9.png">

... it's complaining (and rightly so) about all the various mes I've forced  to exist (probably while trying to figure out issues like this!)
